fx_crash_sig/tests.py

```diff
@@ -1,43 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I moved this to `tests/test_symbolicator.py`, but that's not obvious from the PR changes.